homework/index.js

```diff
@@ -29,8 +29,42 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function renderRepoDetails(repo) {
+    const reposList = document.getElementById('reposList');
+    const repoItem = createAndAppend('li', reposList);
+    const commonBlock = createAndAppend('div', repoItem, {
+      class: 'commonBlock',
+    });
+    const dataTypeList = createAndAppend('ul', commonBlock, {
```

Comment:
Your UI uses two unordered lists, placed side by side, one containing labels and the other containing the repository values. When viewed on a mobile phone the long repository values are clipped:

![image](https://user-images.githubusercontent.com/2788771/65312235-60f6b580-db92-11e9-8e2b-86efd0e1911c.png)

This is undesired. But if you somehow manage to let the repository values wrap over multiple lines you run into another problem because of your chosen organisation. Wrapped lines for the description will push the values for **Forks** and **Updated** down and they will no longer align with their labels.

Can you try another structure of your HTML that solves these problems, perhaps an HTML table?

Reply:
I have changed all of it.